pyanaconda/modules/common/constants/objects.py

```diff
@@ -115,3 +116,12 @@
     namespace=PAYLOADS_NAMESPACE,
     basename="Packages"
 )
+
+# System services
+
+# Subscription objects.
+
+RHSM_CONFIG = DBusObjectIdentifier(
+    namespace=RHSM.namespace,
```

Comment:
Please, use `RHSM_NAMESPACE`.

Reply:
OK, will do - I used this as we at the moment don't have any name spaces defined for non-Anaconda DBus services.